tests/test_io_xarray.py

```diff
@@ -321,3 +322,53 @@ def test_xarray_reader_resampling():
 
         with pytest.warns(DeprecationWarning):
             _ = dst.feature(feat, resampling_method="nearest")
+
+
+def test_xarray_reader_no_crs():
+    """Should raise MissingCRS."""
+    arr = numpy.arange(0.0, 33 * 35).reshape(1, 33, 35)
+    data = xarray.DataArray(
+        arr,
+        dims=("time", "y", "x"),
+        coords={
+            "x": list(range(-170, 180, 10)),
+            "y": list(range(-80, 85, 5)),
```

Comment:
just a nitpick - why not

```suggestion
            "x": numpy.arange(-170, 180, 10),
            "y": numpy.arange(-80, 85, 5),
```